setup/Tycho.setup

```diff
@@ -305,13 +317,6 @@
           rootFolder="${git.cloneTycho.location}/tycho-testing-harness"/>
       <description>Import Maven Project</description>
     </setupTask>
-    <setupTask
```

Comment:
Why not import integration tests?

Reply:
The integration tests have just been moved upwards because they were the only import not in alphabetical order.